.github/workflows/build.yml

```diff
@@ -701,12 +701,15 @@ jobs:
           || (github.ref == 'refs/heads/rc' && needs.setup.outputs.hotfix_branch_exists == 0)
           || github.ref == 'refs/heads/hotfix-rc'
         env:
-          APPLE_ID_USERNAME: ${{ secrets.APPLE_ID_USERNAME }}
-          APPLE_ID_PASSWORD: ${{ secrets.APPLE_ID_PASSWORD }}
-        run: |
-          xcrun altool --upload-app --type ios --file "./bitwarden-export/Bitwarden.ipa" \
-              --username "$APPLE_ID_USERNAME" --password "$APPLE_ID_PASSWORD"
-
+          APP_STORE_CONNECT_TEAM_ISSUER: ${{ secrets.APP_STORE_CONNECT_TEAM_ISSUER }}
+          APP_STORE_CONNECT_AUTH_KEY: 6TV9MKN3GP
```

Comment:
❗Should the auth key be here in plain text? Isn't that a confidential value?

Reply:
That's how we are using it in `clients` repo. I don't know why @vgrassia made it a plain text

Reply:
That's the key identifier and not the actual .p8 key content. This PR is missing setting up that [file](https://github.com/bitwarden/clients/blob/8d507c04a03fec69ed4ca23ae68669bcb0da096b/.github/workflows/build-desktop.yml#L786-L791). 

Any reason why we're not using the [key](https://github.com/bitwarden/ios/blob/main/.github/workflows/build.yml#L366) used in the iOS repo?

Reply:
I'm updating that in a minute @vvolkgang, I used the Desktop key unkowing it's only for Desktop

Reply:
Got it. Thanks for the explanation. ☺️

Reply:
@vvolkgang We use a separate key per repository for security.